README.md

```diff
@@ -0,0 +1,71 @@
+
+# EMC_post
+
+The EMC_post package provides tools for post processing for NCEP models.
+
+This is part of the [NCEPLIBS](https://github.com/NOAA-EMC/NCEPLIBS)
+project.
+
+## Authors
+
+NCEP/EMC Developers
+
+Code Manager: WenMeng
+
+## Prerequisites
+
+This package requires the following NCEPLIBS packages:
+
+- [NCEPLIBS-g2](https://github.com/NOAA-EMC/NCEPLIBS-g2)
+- [NCEPLIBS-g2tmpl](https://github.com/NOAA-EMC/NCEPLIBS-g2tmpl)
+- [NCEPLIBS-sp](https://github.com/NOAA-EMC/NCEPLIBS-sp)
+- [NCEPLIBS-ip](https://github.com/NOAA-EMC/NCEPLIBS-ip)
+- [NCEPLIBS-bacio](https://github.com/NOAA-EMC/NCEPLIBS-bacio)
+- [NCEPLIBS-w3emc](https://github.com/NOAA-EMC/NCEPLIBS-w3emc)
+- CRTM
+
+Also required to build NCEPpost executable (cmake option
+BUILD_POSTEXEC):
+
+- [NCEPLIBS-sigio](https://github.com/NOAA-EMC/NCEPLIBS-sigio) -
+- [NCEPLIBS-sfcio](https://github.com/NOAA-EMC/NCEPLIBS-sfcio) -
+- [NCEPLIBS-nemsio](https://github.com/NOAA-EMC/NCEPLIBS-nemsio) -
+- [NCEPLIBS-gfsio](https://github.com/NOAA-EMC/NCEPLIBS-gfsio)
+
+
+The [NCEPLIBS-wrf_io](https://github.com/NOAA-EMC/NCEPLIBS-wrf_io)
+library is required to build with NCEPpost with WRF-IO library (cmake
+option BUILD_WITH_WRFIO).
+
+The following third-party libraries are required:
```

Comment:
What about jasper, zlib, and png?  I'm just getting familiar with nceplibs package and the cmake build, so apologies if this is obvious.  These are taken care of through nceplibs-external, correct? So do they not need to be listed here if using nceplibs as a dependency for cmake build of upp?

Reply:
jasper, zlib, and png are not checked for in CMakeLists.txt. Are they required to build EMC_POST? If so, they should be listed here. Also they should probably be checked for in the top-level CMakeLists.txt.

Reply:
Actually good point. They have been requirements for so long it is engrained in my head.  But we do not require them for community build as of the most recent release. 
@WenMeng-NOAA - Can you comment/confirm the requirement of jasper, zlib, png for any operational needs?

Reply:
Yes, these three are required for operational implementation.

Reply:
I might think w3nco is required too.

Reply:
OK, so should these be listed as requirements?

If so, should they be checked for in the CMake build?

Reply:
Yes.